### ISSUE TITLE
Git command to host a repo

### ISSUE THREAD

**Issue by AustinFoss** [isomorphic-git/isomorphic-git#1697]
Let me know what git commands aren't implemented yet that you really miss, or any other ideas you have!

**Comment by Qu4tro** [comment]
I don't see an action point for this.

`isomorphic-git` is a git client. Having it work as a nodejs based server, would be a big project, without a lot of support as we'll be competing directly with git and other git servers, like gitea.

I suggest this issue to be closed @jcubic and labeled wontfix.

**Comment by AustinFoss** [comment]
Yeah I agree. More than a year later rereading this I failed tremendously making this clear 

I may remake a new issue with proper descriptions of what I actually meant by it one day.

I remember what I was intending when I wrote it; was exploring how to implement this example project using a purely web based front end: https://docs.ipfs.tech/how-to/host-git-repo/

The commands in question I meant to reference were `git update-server-info` & `git unpack-objects`.

Again, feel free to close this issue. It was not clear at all, and I don't have the bandwidth to pursue this currently.

To be clearer though, this wouldn't mean iso-git was acting as a Git server on its own. These commands would simply prep an existing repo to be deployed to a "dumb server"

From the docs: "A dumb server that does not do on-the-fly pack generations must have some auxiliary information files in $GIT_DIR/info and $GIT_OBJECT_DIRECTORY/info directories to help clients discover what references and packs the server has. This command generates such auxiliary files."

The ipfs service acts as the dumb server, while iso-git would simply do the cloning, and editing.
